modules/sdk-api/test/unit/encrypt.ts

```diff
@@ -0,0 +1,71 @@
+import * as assert from 'assert';
+import { randomBytes } from 'crypto';
+
+import { decrypt, encrypt } from '../../src';
+
+describe('encryption methods tests', () => {
+  describe('encrypt', () => {
+    it('encrypts the plaintext with the given password', () => {
+      const password = 'myPassword';
+      const plaintext = 'Hello, World!';
+      const ciphertext = encrypt(password, plaintext);
+
+      assert(ciphertext !== plaintext, 'ciphertext should not be equal to plaintext');
+    });
+
+    it('returns a different ciphertext for the same plaintext and password', () => {
+      const password = 'myPassword';
+      const plaintext = 'Hello, World!';
+      const ciphertext1 = encrypt(password, plaintext);
+      const ciphertext2 = encrypt(password, plaintext);
+
+      assert(ciphertext1 !== ciphertext2, 'ciphertexts should not be equal');
+    });
+
+    it('throws an error if the salt length is not 8 bytes', () => {
```

Comment:
Do we have docs somewhere that says it needs to be 8 bytes?

Reply:
Its a requirement we added, nothing new in my PR, was already there.